docs/examples.md

```diff
@@ -143,15 +143,38 @@ To change the language translation, you can set the language explicitly.
 
 ## Holiday categories support
 
-To get a list of other categories holidays (for countries that support them):
+The holidays library supports various categories to classify different types of holidays. This allows you to filter holidays based on their official status, religious significance, or institutional relevance.
+
+### Basic Category Usage
+
+Categories accept a single string (e.g., 'public') or an iterable of strings, but it's recommended to use constants from `holidays.constants` for better maintainability; matching is case-sensitive.
+
+To get holidays from specific categories:
 
 ``` python
->>> for dt, name in sorted(holidays.BE(years=2023, language="en_US", categories=BANK).items()):
->>>     print(dt, name)
-2023-04-07 Good Friday
-2023-05-19 Friday after Ascension Day
-2023-12-26 Bank Holiday
+>>> import holidays
+>>> from holidays.constants import PUBLIC, UNOFFICIAL
+>>> # Get only public holidays
```

Comment:
Code comments like this should end with a dot (`.`) - same goes for other entries 👍 
```suggestion
>>> # Get only public holidays.
```